app/code/Magento/Security/Model/AdminSessionInfo.php

```diff
@@ -133,7 +134,11 @@ public function isSessionExpired()
         $currentTime = $this->dateTime->gmtTimestamp();
         $lastUpdatedTime = $this->getUpdatedAt();
         if (!is_numeric($lastUpdatedTime)) {
-            $lastUpdatedTime = strtotime($lastUpdatedTime);
+            if ($lastUpdatedTime === null) {
+                $lastUpdatedTime = 0;
+            } else {
+                $lastUpdatedTime = strtotime($lastUpdatedTime);
+            }
         }
 
         return $lastUpdatedTime <= ($currentTime - $lifetime) ? true : false;
```

Comment:
```suggestion
        return $lastUpdatedTime <= ($currentTime - $lifetime);
```

Reply:
this remove make no sense but as return will be either true or false, to optimize point of view its good to remove